wix/license.rtf

```diff
@@ -0,0 +1 @@
+{\rtf1}
```

Comment:
I'm not familiar with wix, but Alacritty's license should probably be put here.

Reply:
It displays during installation and asks the user to accept the agreement. Should I copy and paste the license from [here](https://github.com/jwilm/alacritty/blob/master/LICENSE-APACHE)?

Reply:
Yes, please.